### DISCUSSION TITLE
What does $t mean in the default status field choices?

### DISCUSSION THREAD

**Discussion by AaronBeaudoin** [directus/directus#12861] [Archive]
In the default choices for the "status" field, you get this, for example:

![image](https://user-images.githubusercontent.com/26592486/164074065-3d48bba9-0c01-4e4e-8f83-e416fedfdddd.png)

I found the documentation for [Dynamic Variables](https://docs.directus.io/configuration/filter-rules/#dynamic-variables), which I think is related to this somehow, but `$t` isn't listed.

**So here are my questions:**

- What does `$t` mean and what does it do?
- Are there other undocumented dynamic variables?
- Is there a way to add custom dynamic variables?

**Comment by rijkvanzanten** [comment]
It uses the translated key values! If you click that translations icon in the right of the input, you can make your own keys with custom translations 🙂 

@azrikahar It should be highlighted in the same purple though, so maybe we can improve on that
